app/src/main/java/net/nhiroki/bluelineconsole/applicationMain/CandidateListAdapter.java

```diff
@@ -117,6 +123,7 @@ public View getView(final int position, View convertView, @NonNull ViewGroup par
     public void notifyDataSetChanged() {
         this.unrecyclableViews.clear();
         this.chosenNowExplicitly = CHOICE_NOT_SET_YET;
+        shall_hide_icon = PreferenceManager.getDefaultSharedPreferences(getContext()).getBoolean("pref_appearance_hide_icon",false);
```

Comment:
This is not a time that suggests this configuration is changed.

Maybe it may work for most cases, but at least it needs a good explanation why you place this code here.

Actually I think there may be better place.